### PR TITLE
Conner Johnson Checkpoint-5

### PR DESCRIPTION
**Pull request by cconnerj**

### CODE REVIEW

**Comment by cconnerj** [comment]
completness: 4
comfort: 5

File: todo_app/app/views/todos/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<h1>TO-DO list</h1>
+
+<ul>
+  <% @todos.each do |todo| %>
+  <li>
+    <a href="/todos/<%= todo.id %>">
+      <%= todo.content %>
+    </a>
```
Comment (jsm13) [inline todo_app/app/views/todos/index.html.erb:8]: could use `<%= link_to todo.content todo_path(todo) %>`

File: todo_app/db/seeds.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+#
+#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
+#   Character.create(name: 'Luke', movie: movies.first)
+
+Todo.destroy_all
+
+take_out_trash = Todo.create(content: "Take trash to the curb", is_completed: false)
+pay_bills = Todo.create(content: "Pay water and gas bill", is_completed: false)
+feed_dog = Todo.create(content: "Feed dog at 8 am", is_completed: false)
+wake_up = Todo.create(content: "Wake up at 7am", is_completed: false)
```
Comment (jsm13) [inline todo_app/db/seeds.rb:14]: Don't need local variables if you aren't using them to relate things

**Review by jsm13 — COMMENTED** [review]
Excellent job!

**Pull request closed without merging**